cmd/querier/main.go

```diff
@@ -88,6 +88,7 @@ func main() {
 
 	subrouter := server.HTTP.PathPrefix("/api/prom").Subrouter()
 	subrouter.PathPrefix("/api/v1").Handler(middleware.AuthenticateUser.Wrap(promRouter))
+	subrouter.Path("/remote_read").Handler(middleware.AuthenticateUser.Wrap(http.HandlerFunc(queryable.Q.RemoteReadHandler)))
```

Comment:
`pull` sounds a bit funny though, no? :) But ok if you think we should take that.

Ah right, it works through my local authfe, but I pointed my Prom to `/api/prom/remote_read`, not `/prom/remote_read`. Will add that to authfe once we know what name we want.